### ISSUE TITLE
(Dynamic) Favorites depending on time or wifi network.

### ISSUE THREAD

**Issue by bioRex21** [8bitPit/Niagara-Issues#165]
Being able to have a different set of favorites (Slack, email, Skype, Calendar, etc) while at work. And a different set at home (Netflix, Twitter, etc). 
Could also be depending on the time.

**Comment by 8bitPit** [comment]
Yeah, would be great. In the future I'll experiment with the Google Awareness API.

**Comment by vivekkrish** [comment]
This is a great suggestion!

@8bitPit, please do look at this app/widget called Contextual App Folder (C.A.F), https://play.google.com/store/apps/details?id=com.ctxwidget, which implements functionality similar to the OPs suggestion.

C.A.F is actually my most favorite app to use on the home screen, but it seems to unfortunately not be under development any more. Would love to see some of it's features ported into your wonderful launcher.

**Comment by DGollings** [comment]
Or bluetooth connection. Specifically when in car and you want car specific apps with as few clicks as possible

**Comment by disacrol** [comment]
Location based would be awesome too.

**Comment by disacrol** [comment]
@8bitPit also make sure that apps that will be shown when Bluetooth is connected are not considered in the most used algorithm when the user taps to open them. The user would get access to them at that situation and probably wouldn't want them to show on favorites all the time.

**Comment by 8bitPit** [comment]
This won't be implemented in Niagara Launcher itself in the near future, sorry. However, I plan to add an API for [Tasker](https://play.google.com/store/apps/details?id=net.dinglisch.android.taskerm&hl=de):  #786 
This app is super powerful and let's you make triggers for almost everything (See its description on Google Play: _Automate stuff based on the app you're in, the time of day, your location, your Wi-Fi network, Received SMS or Calls, the currently playing song and many other (130+) states and events_)

**Comment by 8bitPit** [comment]
To better plan ahead and to give users an estimate which features will be added to Niagara, I’ve decided to close all feature requests that that are unlikely to be implemented in the next year (2020). This is one of those, unfortunately. I'm sorry. Feel free to open this ticket again in 2021 or later.

**Comment by TFackler** [comment]
Since we keep getting requests for this feature, I will re-open it.

**Comment by TFackler** [comment]
From Trevor via Mail support:

> I would like to be able to do the following
>
> Promote a collection of apps/shortcuts in the morning and in the evening. And perhaps demote other apps.
>
> Promote a folder when connected to specific wifi or Bluetooth (eg my house or my car).
>
> The stock pixel launcher's suggested apps is one area that gets this right for me most of the time. For example, it highlights my alarm app before bed because I usually adjust/check my alarm.

**Comment by ssssamstrup** [comment]
I thought this would be the right place to put this. Let me know if you want me to create a seperate request for my idea:

This idea spawned from #537 

I could see the idea of several favorite lists to be useful in a broader sense not just for work apps, with the option of having a favourites lists for just work apps as a toggle.

I know this might go against the idea of simplicity the launcher is aiming for, but having a list for music apps (rather than adding them dynamically to the favorites list) would be our useful for a lot of users; at lest it seems so from how many times dynamic favorite lists have been suggested. This would also make powerusers able to access more favourites in a minimalistic way without scrolling.

If done with some conditional settings (work hours)/automation, changing the default shown list based on which list you use most at what time of day fx. or as just for work apps based on user input work hours. I could see this being done for morning/afternoon/evening apps, workout apps, focus apps (could be part of your wellbeing initiative), etc.

I know a lot of this could be done with tasker integration (which I applaud), but for powerusers like me who aren't programmers, tasker is confusing and setting up several different conditions to work with each other in Tasker... I've already given up at just the thought of the though.

I think creating a strong platform within Niagara for you (the team) to implement conditional favorites/homescreens could be great in the long run, for the powerful simplicity it would bring for your users. I also believe it's important to spend some time on implementing the right way. 😊

**Comment by shbumba** [comment]
@ssssamstrup I just jumped in this topic because I wanted to make the same feature request, I had an idea closer to your vision!

Dear Niagara team, I'd love to have something similar to what described above. In my use case I'd like to have an ability to switch between different profiles on my choose. Like, when I want to study on learn something I would like to select one of the pre-prepared profiles (by me) e.g. a "Study" profile that will change the default favorite list to the one I preconfigured as  for this profile. The same way I'd like to switch between other different profiles like: "work", "sport", "resting", "focus" etc.

In my vision this might look like a list of tags positioned horizontally above of the apps list.

I've seen so many requests for this feature, that I'm not giving up hope of seeing it soon it your roadmap. Thx you for your work!
